addons/calendar/models/calendar.py

```diff
@@ -529,6 +529,14 @@ class Meeting(models.Model):
     _inherit = ["mail.thread"]
 
     @api.model
+    def default_get(self, fields):
+        res = super(Meeting, self).default_get(fields)
+        if self.env.context.get('active_model') and self.env.context.get('active_id') and not self.env.context.get('active_model') == 'mail.activity':
+            res['res_id'] = self.env.context['active_id']
+            res['res_model'] = self.env.context['active_model']
```

Comment:
You should set res_model_id, not res_model. res_model is a readonly related field and we should not try to write on it.